recipes/python-qpot/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "python-qpot" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "0ecb77f5e4cab70a49946f9ecc3e32640e64ab4b0e148c9ccd67cf46006174db" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/QPot/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```

Comment:
You should have make and cmake as deps here

Reply:
Here I use pure Python to compile. I don't think I should add make and/or cmake as additional deps